### ISSUE TITLE
[urgent and also very easy] Remove --pool-type from chain recipes and any other recipes where it's found.

### ISSUE THREAD

**Issue by danpovey** [kaldi-asr/kaldi#860]
Below is just an example of where we have the outdated --pool-type strings in some chain recipes (issue pointed out by Mike Newman).  These options have been removed from the scripts to where they are passed, and anyway have default values, so they should be removed.
```
a09:egs: cd swbd/s5c
a09:s5c: grep pool-type steps/*/*.sh
a09:s5c: grep pool-type local/*.sh
a09:s5c: grep pool-type local/*/*.sh
local/chain/run_tdnn_4q.sh:    --pool-type "$pool_type" \
local/chain/run_tdnn_6v.sh:  pool_opts=$pool_opts${pool_type:+" --pool-type $pool_type "}
local/chain/run_tdnn_6w.sh:  pool_opts=$pool_opts${pool_type:+" --pool-type $pool_type "}
local/chain/run_tdnn_6x.sh:  pool_opts=$pool_opts${pool_type:+" --pool-type $pool_type "}
local/chain/run_tdnn_6y.sh:  pool_opts=$pool_opts${pool_type:+" --pool-type $pool_type "}
local/chain/run_tdnn_6z.sh:  pool_opts=$pool_opts${pool_type:+" --pool-type $pool_type "}
local/chain/run_tdnn_7a.sh:  pool_opts=$pool_opts${pool_type:+" --pool-type $pool_type "}
local/chain/run_tdnn_7b.sh:  pool_opts=$pool_opts${pool_type:+" --pool-type $pool_type "}
local/chain/run_tdnn_7c.sh:  pool_opts=$pool_opts${pool_type:+" --pool-type $pool_type "}
local/chain/run_tdnn_7d.sh:  pool_opts=$pool_opts${pool_type:+" --pool-type $pool_type "}
a09:s5c: 
```

**Comment by vijayaditya** [comment]
I will do this after Interspeech deadline today.

**Comment by danpovey** [comment]
Addressed in PR #995
